projectDocs/translating/crowdin.md

```diff
@@ -0,0 +1,100 @@
+# Translating using Crowdin
+
+Crowdin is used to translate the main NVDA interface.
+NVDA's Crowdin project: <https://crowdin.com/project/nvda>.
+
+This document covers setting up a Crowdin account, connecting it with PoEdit, and translating the main interface using Crowdin and PoEdit.
+
+## Setup
+
+### Create Crowdin account
+
+1. Create a [Crowdin account](https://accounts.crowdin.com/register?continue=%2Fproject%2Fnvda).
+1. Message the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org> to request being added as a translator.
+Please include your Crowdin username and the languages you wish to translate.
+
+#### Translation reviews
+
+Translated strings will need to be reviewed and approved by a proofreader before being included in NVDA.
+Translators may have proofreader status granted on a case-by-case basis by messaging the above contact points.
+
+### Setup PoEdit
+
+It is recommended that you use the latest version of PoEdit and NVDA for translating.
+Alternatively, you can use Crowdin's interface directly.
+
+PoEdit supports connecting with Crowdin directly.
+PoEdit's homepage is: <http://www.poedit.net/>
+
+1. Download the latest Windows PoEdit version at <https://poedit.net/download>
+1. Install it by following the on-screen instructions, the default options should be sufficient.
+1. When launching PoEdit:
+    1. Choose "Translate cloud project"
+    1. Connect your Crowdin account
+    1. Select NVDA and the language you wish to translate
+
+## Translating using PoEdit

```

Comment:
The following information is missing and should be added in this paragraph:
* some word have plural forms; describe how to manage it, especially with poedit in which object navigation is required to reach the plural form
* some string have context information (when `pgettext` is used in the code)

Reply:
I've added new sections here, let me know what you think.